src/signals/incident/components/IncidentPreview/components/Map/__snapshots__/index.test.js.snap

```diff
@@ -30,7 +30,7 @@ exports[`Preview component <Map /> should render only address correctly 1`] = `
       >
         <div>
           <div>
-            Hell 666C
+            Hell 666C, undefined undefined
```

Comment:
The test data is incomplete, if undefined is shown here?

Reply:
scherp! ja verbeterd